modules/fedimint-ln-server/src/lib.rs

```diff
@@ -1578,11 +1582,22 @@ mod fedimint_migration_tests {
                                 .await;
                             let num_shares = proposed_decryption_shares.len();
                             ensure!(
-                            num_shares > 0,
-                            "validate_migrations was not able to read any ProposeDecryptionShares"
-                        );
```

Comment:
The `ensure!` macro seems to give `cargo fmt` issues that go unnoticed. Has anyone bumped into other issues with `cargo fmt` silently failing?

Reply:
A lot of macros have that problem (e.g. `stream!` and `select!`).